index.html

```diff
@@ -1074,9 +1086,11 @@
                 this.balls[i].render();
             }
             ctx.restore();
+
+            allBallsAreStationary &= Math.abs(this.balls[i].velocity.x) < PREFS.showStickAt && Math.abs(this.balls[i].velocity.y) < PREFS.showStickAt;
```

Comment:
@oyvindjam nice job on this PR.  I see this line twice in this PR.  Can you make this a simple helper function?